packit/config/config.py

```diff
@@ -161,7 +161,7 @@ def load_authentication(raw_dict):
             )
             pagure_user_token = raw_dict.get("pagure_user_token")
             pagure_instance_url = raw_dict.get(
-                "pagure_instance_url", "https://src.fedoraproject.org"
+                "pagure_instance_url", "https://src.stg.fedoraproject.org"
```

Comment:
We shouldn't overwrite this. This will force `stage` Pagure also for prod users.

We should use `authentication` key for different instances to not need to handle them manually.